genomic_data_service/rnaseq/domain/constants.py

```diff
@@ -0,0 +1,62 @@
+AT_ID = '@id'
+
+AT_TYPE = '@type'
+
+BASE_PATH = '/tmp/'
+
+DATASET = 'dataset'
+
+DATASETS = 'datasets'
```

Comment:
I believe these variables won't change so soon. This pattern is getting a bit overused in snovault, should we cut a few of these?

Reply:
Valid point but will defend a little. Like this pattern because it makes clear all the places that rely on this one variable (versus other possible ambiguous uses of the string 'datasets'). It's also great for tab completion and in this case I think a lot safer since it's used as a key for getting and setting a dictionary value.

Reply:
Oh my comment didn't refer to datasets only, but to the list, sorry should have marked all lines. No problem, we can keep the ones used multiple times, but in cases like:

https://github.com/ENCODE-DCC/genomic-data-service/pull/4/files#diff-ae5121cf74b5d379c26e8ca5dbeb58eb667eea412c65fe540d58919ae9698741R138-R143

if we decide to apply, then all keys must turn into constants and it can get too hard to read. Or if we don't we are defining the constants and not using them.

Reply:
Ah, you're not wrong. Don't feel as strongly about that example since it's a more self-contained building up of a dictionary. Maybe mixing and matching instead of being 100% consistent either way is okay?

Reply:
as long as it doesn't get out of control, it's all good haha